### PR TITLE
Fixes #1 - Incorporate Table 6

### LINKED ISSUE #1: Incorporate Table 6
The `weightEst2` function needs to take in the following parameters: x1, x2, SEP1, SEP2, region (ex. East-Central Plains hydrologic region), equation method (ex. rBC,Wac), and AEP (ex. Q50, which comes from the equation). 

The `weightEst3` function needs to take in the following parameters: x1, x2, SEP1, SEP2, region (ex. East-Central Plains hydrologic region), 3 equation methods (ex. rBC,Wac, rBC,Wbf, rBC,Wrs), and AEP (ex. Q50, which comes from the equation). 

Then, create a helper method that will take the region, equation method, and AEP to look up the `r12`, `r13`, and `r23` values from Table 6 in this report: https://pubs.usgs.gov/sir/2020/5142/sir20205142.pdf. Table 6 values will be hard-coded.

### PR DESCRIPTION
**Pull request by amedenblik**
Closes #1 

- Created coefficient_table.py with Table 6 values from the report: https://pubs.usgs.gov/sir/2020/5142/sir20205142.pdf
- Created hydrologic_region_table.py to translate Regression Region codes to the hydrologic regions from Table 6
- Added `getCrossCorrelationCoefficient` function to extract the correlationCoefficient from Table 6
- Edited `weightEst2` and `weightEst2` functions to work with the new `getCrossCorrelationCoefficient` function
- Edited documentation

### CODE REVIEW

**Review by harper-wavra — APPROVED** [review]

**Comment by aaronstephenson** [comment]
This looks great! The only thing I would change would be to exclude the entire `__pycache__` folder (which contains binary .pyc files).

**Review by aaronstephenson — CHANGES_REQUESTED** [review]
Jus remove the `__pycache__` folder (also consider adding it to your .gitignore file) and this is good to go.

File: README.md [hunk 1]
```diff
@@ -18,31 +16,32 @@ To run the services locally, run the following in your Windows command prompt:
 # create a virtual environment
 python -m venv env
 # active the virtual environment
-. env/Scripts/activate
+.\env\Scripts\activate
 # install the project's dependencies
 pip install -r requirements.txt
 # deploy at a local server
-uvicorn Main:app --host 0.0.0.0 --port 8000
+uvicorn main:app --host 0.0.0.0 --port 8000 --reload
```
Comment (harper-wavra) [inline README.md:23]: Should we get rid of reload?

Comment (aaronstephenson) [reply README.md:23]: I think it depends. For development, it's very useful, but you absolutely do not want it for a server deployment. Maybe mention both scenarios in the README?

**Review by harper-wavra — APPROVED** [review]

**Pull request merged**